### PR TITLE
Milestone 1: Add classes

### PR DESCRIPTION
**Pull request by Rachelwebdev**
### In this Milestone, I;

- Created a class `Person` with getters, setters, and methods

- Created a class `Student` which inherits from `Person` and has a method `play_hooky`

- Created a class `Teacher`which inherits from `Person`

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @Rachelwebdev,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Your linter checks failed and you do not have a descriptive readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
```
Comment (danmainah) [inline person.rb:18]: - [ ] @Rachelwebdev great work on your method however the name is wrong kindly change it to `is_of_age` as it is in the project description
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Rachelwebdev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: person.rb (hunk 1 above)
Comment (Rachelwebdev) [reply person.rb:18]: I tried using that but I got linter errors to change it to of_age

**Review by Ismailco — APPROVED** [review]
### Hi @Rachelwebdev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rachelwebdev** [comment]
Thanks for the approval

**Pull request merged**